README.md

```diff
@@ -24,8 +57,27 @@ processed on startup with environment variables passed in via the docker run
 command-line or via your docker-compose manifest file. Here are the "tunable"
 configurations offered by this image.
 
+This only applies to Node v4 and v6 as examples of the implicit "Docker Image Version 1"
+variant of our docker-node implementation.
+
 * `SKIP_NPM_INSTALL`: If this is set (to anything) it will not run `npm install` when the container boots.
 
+## Version 1 vs. Version 2
```

Comment:
Do you think there is enough usage of those images, and the changes to not use those, are significant enough to warrant that versioning scheme?  That seems like an Outriggerism that could cause confusion.

Reply:
Per discussion, we've decided not to have two versions. The question is now whether we want to have an explicit v1.